nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -90,46 +90,74 @@ def _add_suffix(params, model):
 
 def _get_file_name(nii_file):
     """Construct the raw confound file name from processed functional data."""
+    import itertools
```

Comment:
any reason this is not imported at the top of the module?

Reply:
No good reason. I'll move it.